lib/sensors/low_power_bms.cpp

```diff
@@ -0,0 +1,43 @@
+#include "low_power_bms.hpp"
+
+namespace hyped::sensors {
+
+std::optional<LowPowerBMS> LowPowerBMS::create(core::ILogger &logger,
+                                               std::shared_ptr<io::II2c> i2c,
+                                               const std::uint8_t device_address)
+{
+  if (device_address != kDefaultLpBmsReadAddress) {
+    logger.log(core::LogLevel::kFatal, "Invalid device address for accelerometer");
+    return std::nullopt;
+  }
```

Comment:
Are there no other checks to be made?

Reply:
imo this isn't needed here. if there's something that checks the health of the BMS that would be good instead.